File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/SeatQuantity.cs
using System;

namespace Registration
{
    [Serializable]
    public class SeatQuantity
    {
        public SeatType Seat { get; private set; }
        public int Quantity { get; private set; }

        public SeatQuantity() { }
        public SeatQuantity(SeatType seat, int quantity)
        {
            Seat = seat;
            Quantity = quantity;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/SeatQuantity.java
package com.microsoft.conference.registration.domain;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class SeatQuantity {
    private SeatType seatType;
    private int quantity;

    public SeatQuantity() {
    }

    public SeatQuantity(SeatType seatType, int quantity) {
        this.seatType = seatType;
        this.quantity = quantity;
    }
}
